src/components/StyledSwipeableDrawer/StyledSwipeableDrawer.tsx

```
@@ -46,7 +46,12 @@ export default function StyledSwipeableDrawer({
       allowSwipeInChildren
     >
       {!open && isMobileWidth && (
-        <div className={styles['drawer-puller']} data-testid={`${testId}-drawer-puller`} />
+        // eslint-disable-next-line jsx-a11y/click-events-have-key-events
+        <div
+          className={styles['drawer-puller']}
+          onClick={onOpen}
+          data-testid={`${testId}-drawer-puller`}
+        />
```

ESLint

[error] Avoid non-native interactive elements. If using native HTML is not possible, add an appropriate role and support for tabbing, mouse, keyboard, and touch inputs to an interactive content element.

(jsx-a11y/no-static-element-interactions)